### ISSUE TITLE
s3dir s3 path completion broken

### ISSUE THREAD

**Issue by tilusnet** [aboisvert/s3cp#16]
It only provides the verbatim. Twice.

I am typing
`s3dir s3://bucket/a<TAB><TAB>`
and I get
`//bucket/a //bucket/a`

**Comment by aboisvert** [comment]
Yes, sorry about that!   The command-line completion was experimental and basically left unfinished.  I don't use bash shell much nowadays and so never completed the work there.  I didn't help that I was a newbie at bash command-line completion and that bash' documentation (and related info on the web) on command-line completion was lacking.   The learning curve was not good...

**Comment by aboisvert** [comment]
If you're interested in contributing something here, please do!  I would welcome any further improvements.  I do not plan to improve things myself (except perhaps adding support for command-line completion for fish shell someday)

**Comment by tilusnet** [comment]
Thanks for explaining it. I am not familiar with shell completion either even less so with Ruby.
Perhaps someone else reading this is able to take on this task with much more ease.
